src/Symfony/Component/DependencyInjection/Compiler/ResolveInstanceofConditionalsPass.php

```diff
@@ -113,7 +113,7 @@ private function processDefinition(ContainerBuilder $container, string $id, Defi
                 $definition = substr_replace($definition, 'Child', 44, 0);
             }
             $definition = unserialize($definition);
-            /** @var ChildDefinition $definition */
+            /* @var ChildDefinition $definition */
```

Comment:
this looks weird. I think this should be solved by putting the `@var` phpdoc on the assignment statement instead.